src/components/Navbar/Navbar.tsx

```diff
@@ -15,20 +14,21 @@ import {
   Toolbar,
   Typography,
   useMediaQuery,
+  useTheme,
 } from '@mui/material';
 import Image from 'next/image';
 import { useState } from 'react';
 import LogoImage from '../../assets/images/mainLogo.png';
 
-export const Navbar = () => {
+function Navbar() {
```

Comment:
Please revert this change. We may be passing a link array directly into the nav bar later.